### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by nopa-2**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <tite>Welcome to Potionfy</tite>
 </head>
 <body>
-  
+  <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (nopa-1) [inline index.html]: Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?

Comment (nopa-2) [reply index.html]: Sure can do!

**Review by nopa-1 — CHANGES_REQUESTED** [review]
Please change the code as per my requests on the review.

**Review by nopa-1 — APPROVED** [review]
Well done!

**Pull request merged**